src/Model/Customer.php

```diff
@@ -242,7 +241,7 @@ public function jsonSerialize()
             'email_address' => $this->emailAddress,
             'name' => $this->name,
             'phone_number' => $this->phoneNumber,
-            'customer_since' => $this->customerSince ? $this->customerSince->format(DateTime::ISO8601) : '',
```

Comment:
Not 100% sure about this. If I remember correctly the format is not 100% the same. 
Just tested:

```php
$date = new DateTime();
echo $date->format(DateTimeInterface::ISO8601) . PHP_EOL;
echo $date->format(DateTimeInterface::ATOM) . PHP_EOL;
```

Outputs:

```
2024-09-17T08:54:01+0000
2024-09-17T08:54:01+00:00
```

Not exactly sure what is expected, but better safe than sorry I would suggest to leave the deprecated format.

Reply:
Reverted all changes referencing ATOM back to ISO8601.